modules/functions.js

```diff
@@ -0,0 +1,79 @@
+const booksLibrary = document.querySelector('#booksLibrary');
+const form = document.querySelector('#newBook');
+const booksTable = document.querySelector('#booksTable');
+const contact = document.querySelector('#contact');
+export function colorCards() {
```

Comment:
- [ ] As per the project requirement kindly ensure to **replace all regular functions with arrow functions**